html-and-web/html-and-the-web_div-salad/index.html

```diff
@@ -7,65 +7,56 @@
 
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>HTML Basics Excercise</title>
-    <meta charset="UTF-8" />
-  </head>
-  <!--someone forgot to wrap the content of our webpage with an HTML element,
-	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
-      <!--Which elements could be used to link to parts of the webpage,
-		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
-    <!--Looks like the following part is the central content of our webpage.-->
-    <div>
-      <div id="about-me">
-        <div>About me</div>
-        <div>
+  <body>
+    <head>
+      <title>HTML Basics Excercise</title>
+      <meta charset="UTF-8" />
+    </head>
+    <header>
+      <h1>Jane Doe</h1>
+      <nav id="nav">
```

Comment:
you could add the IDs of the sections after the # to make the navegator (i.e. #about-me)